opteryx/results/c6a.4xlarge.json

```diff
@@ -0,0 +1,57 @@
+{
+    "system": "Opteryx",
+    "date": "2025-01-03",
+    "machine": "c6a.4xlarge, 500gb gp2",
+    "cluster_size": 1,
+
+    "tags": ["Python", "column-oriented", "ad-hoc"],
```

Comment:
"column-oriented", "stateless", "serverless", "Parquet" as tags may fit more (I don't find "ad-hoc" and "Python" [here](https://benchmark.clickhouse.com/) already)